tests/data/transforms/test_distance_from_boundary.py

```diff
@@ -0,0 +1,135 @@
+import unittest
+import numpy as np
+import torch
+import matplotlib.pyplot as plt
+
+from ssl4rs.ssl4rs.data.transforms.distance_from_boundary import generate_boundary_distance_mask
+
+class TestGenerateBoundaryDistanceMask(unittest.TestCase):
```

Comment:
the rest of the framework uses pytest, so this should use the same (instead of unittest)